tests/apps/pubsub-publisher/app.go

```diff
@@ -196,11 +485,15 @@ func performPublishHTTP(reqID string, topic string, jsonValue []byte, contentTyp
 }
 
 func performPublishGRPC(reqID string, topic string, jsonValue []byte, contentType string, metadata map[string]string) (int, error) {
+	psName := pubsubName
+	if strings.Contains(topic, bulkSubTopicIdentity) {
```

Comment:
e2e refactor: same as above.

Reply:
Agreed, will do it separately under https://github.com/dapr/dapr/issues/5271. Added over there in the list. Resolving this comment here.